src/index.js

```diff
@@ -1,6 +1,19 @@
 const fibonacci = (n) => {
 
-  }
+  if (n<=1)  return [1];
 
+  let anterior = fibonacci(n - 1);
```

Comment:
This is inefficient. Try to adapt what David did here https://platzi.com/clases/1835-programacion-estocastica/26430-optimizacion-de-fibonacci/ to JavaScript